lib/fluent/output.rb

```diff
@@ -98,7 +98,7 @@ class OutputThread
     def initialize(output)
       @output = output
       @finish = false
-      @next_time = Engine.now + 1.0
+      @next_time = Time.now.to_f + 1.0
```

Comment:
I made change like this to use time object > https://gist.github.com/sonots/3cd09b3022460ec1aac8

But, test did not pass. I felt it makes code complicated. 
Also, note that float + float is faster than time + float.

My opinion is to use float rather than time object. Any opinions? > ALL